armi/reactor/blocks.py

```diff
@@ -2211,12 +2211,21 @@ def autoCreateSpatialGrids(self, systemSpatialGrid=None):
         # Check multiplicities
         mults = {c.getDimension("mult") for c in self.iterComponents()}
 
-        if len(mults) != 2 or 1 not in mults:
-            raise ValueError(
-                "Could not create a spatialGrid for block {}, multiplicities are not 1 or N they are {}".format(
-                    self.p.type, mults
-                )
+        # Do some validation: Should we try to create a spatial grid?
+        multz = {float(m) for m in mults}
+        if len(multz) == 1 and list(multz)[0] == 1.0:
+            runLog.extra(
+                f"Will not create a spatialGrid for block {self.p.type}, multiplicities are are all 1.",
+                single=True,
             )
+            return
+        elif len(multz) != 2 or 1.0 not in multz:
+            runLog.extra(
+                f"Could not create a spatialGrid for block {self.p.type}, multiplicities are not 1 "
+                f"or N they are {mults}",
```

Comment:
We will create the spatial grid if we have a `{1, N}` multiplicity. I think the set notation makes the expectations explicit, and why this block did not support that behavior.
```suggestion
                f"Could not create a spatialGrid for block {self.p.type}, multiplicities are not {{1, N}} "
                f"they are {mults}",
```